incubator/logstash/README.md

```diff
@@ -49,6 +49,7 @@ The following tables lists the configurable parameters of the drone charts and t
 | `elasticsearch.host`   | ElasticSearch hostname                             | `elasticsearch-client.default.svc.cluster.local` |
 | `elasticsearch.port`   | ElasticSearch port                                 | `9200`                                           |
 | `configData`           | Extra logstash config                              | `{}`                                             |
+| `heapSize`             | logstash Heapsize                                  | `1g`                                             |
```

Comment:
I'd replace the description for `heapSize` from "_logstash Heapsize_" to "_Maximum Logstash heap size_"